Tensile/KernelWriterAssembly.py

```diff
@@ -10445,6 +10469,7 @@ def globalWriteElements(self, kernel, vectorWidths, elements,
         if self.ss.cfg.numElementsPerBatchLimitedBySgprs < numElementsPerBatch:
           numElementsPerBatch = self.ss.cfg.numElementsPerBatchLimitedBySgprs
 
+        # TODO: Which of DataType or DestDataType is in a better sense?
```

Comment:
As it looks right now, I would say DestDataType, due to globalWrite? Needs some testing.